app/furniture/marketplace/tag.rb

```diff
@@ -0,0 +1,12 @@
+class Marketplace
+  class Tag < Record
```

Comment:
Not a blocker, but one way to help users keep their tags tidy would be to add a case-insensitive uniqueness constraint here, e.g:

```ruby
validates :label, uniqueness: true, scope: { :bazaar, case_insensitive: true }
```

This would prevent adding "Gluten-Free" if "gluten-free" already exists.

(Also a good excuse to start a smol `tag_spec.rb`...)

Reply:
I like enforcing a case insensitive uniqueness scoped to bazaar!